test/css/samples/general-siblings-combinator-slots-between/input.svelte

```diff
@@ -0,0 +1,14 @@
+<h1>Heading 1</h1>
+<slot>
+  <span>Span 1</span>
+</slot>
+<slot>
+  <span>Span 2</span>
+</slot>
+<p>Paragraph 2</p>
+
+<style>
+  h1 ~ p {
+    color: red;
+  }
+</style>
```

Comment:
for the similar setup, maybe can test for `h1 ~ span` and `span ~ p` ?

Reply:
Good observation! I'll do that, thanks!